ship/src/main/resources/ship-default.conf

```diff
@@ -37,6 +37,85 @@ ship {
 
     # the bucket which contains the import files
     import-bucket = "nexus-ship-import"
+
+    storages {
```

Comment:
It's a bit annoying but in order to create `Storages` we need the full `StoragesConfig` and it requires us to have all of this even if we don't use the local and remote disk

Reply:
Yep when S3 is done and if we need to introduce some support for Azure Blob Storage, we will need to have a look at this.

Reply:
Maybe playing with imports, you could avoid duplication for now ?